Promact.Oauth.Server/src/Promact.Oauth.Server/gulpfile.js

```diff
@@ -27,43 +27,69 @@ paths.concatCssDest = paths.webroot + "css/site.min.css";
 paths.systemConfig = paths.webroot + "systemjs.config.js";
 
 gulp.task("copytowwwroot", function () {
+    var launch = require('./Properties/launchSettings.json');
```

Comment:
launchSettings.json is a specific file for visual studio only. It is not available on deployment server. This will not work.